source/contributing/how-to-migrate.md

```diff
@@ -0,0 +1,34 @@
+# How to Migrate Documentation - Step-by-Step Process

```

Comment:
```suggestion
# Migrating Documentation
```

This is shorter and therefore fits in the sidebar better. There's also a clear sequence in the body of the article so you don't need to include "Step-by-Step Process" in the title.